components/experimental/tests/transliterate/lower_ascii.rs

```diff
@@ -43,58 +48,31 @@ where
     }
 }
 
-#[derive(Debug)]
-struct LowercaseTransliterator(CaseMapperBorrowed<'static>);
-
-impl CustomTransliterator for LowercaseTransliterator {
-    fn transliterate(&self, input: &str, range: std::ops::Range<usize>) -> String {
-        self.0
-            .lowercase_to_string(&input[range], &LanguageIdentifier::default())
-    }
-}
-
 #[test]
 fn test_lower_ascii() {
     let mut collection = RuleCollection::default();
     // Register Latin-ASCII so that the alias mapping gets added
     collection.register_source(
         &"und-t-und-latn-d0-ascii".parse().unwrap(),
-        "<error>".to_string(),
+        "<unused>".to_string(),
         ["Latin-ASCII"],
         false,
         true,
     );
-    // Register Lower so that the alias mapping gets added
-    collection.register_source(
-        &"und-t-und-x0-lower".parse().unwrap(),
-        "<error>".to_string(),
-        ["Any-XLower"],
-        false,
-        true,
-    );
     // Now register our new transliterator
     collection.register_source(
         &"und-t-und-x0-lowascii".parse().unwrap(),
-        // "::NFD; ::[:Nonspacing Mark:] Remove; ::Any-XLower; ::NFC; ::Latin-ASCII;".to_string(),
-        "::NFD; ::[:Nonspacing Mark:] Remove; ::Any-XLower; ::NFC; ::Latin-ASCII;".to_string(),
+        "::NFD; ::[:Nonspacing Mark:] Remove; ::Lower; ::NFC; ::Latin-ASCII;".to_string(),
         [],
         false,
         true,
     );
     let provider = TransliteratorMultiSourceProvider(collection.as_provider());
-    let t = Transliterator::try_new_with_override_unstable(
+    let t = Transliterator::try_new_unstable(
+        &provider,
         &provider,
         &provider,
         &"und-t-und-x0-lowascii".parse().unwrap(),
-        |locale| {
-            if locale.normalizing_eq("und-t-und-x0-lower") {
-                Some(Ok(Box::new(LowercaseTransliterator(
-                    CaseMapperBorrowed::new(),
-                ))))
-            } else {
-                None
-            }
-        },
```

Comment:
Thought: Might be good to retain an example doing this override behavior somewhere, if there isn't already one.

Reply:
Still tested here https://github.com/unicode-org/icu4x/blob/a3bd57daaa7fc2306718ed5bdfd41ecea57d371a/components/experimental/src/transliterate/transliterator/mod.rs#L1477